### ISSUE TITLE
Assets Progress Update #1

### ISSUE THREAD

**Issue by elioqoshi** [lindanlee/tor-media#3]
Hey there,

So I worked all day on the assets today and was able to finish a good chunk of it. This is where I am:

![preview-01](https://cloud.githubusercontent.com/assets/5436686/23096934/db3c7e18-f627-11e6-8b43-ff13fa4e1258.png)

I will create different color variations for most of the assets, in case the classic purple doesn't cut it. In addition, I have added some various sizes for buttons which might come in handy.

I will have to do the Presentation slide templates yet, but that next week probably.

Please ask if you have any questions! 

cc @lindanlee

**Comment by lindanlee** [comment]
Elio:

Good progress on the work! I'm pleased to see all these images coming together. 

Some feedback and questions:  
- do we need the inverted versions of the icons? I would prefer to not have 
![image](https://cloud.githubusercontent.com/assets/6580269/23173671/8f678498-f81f-11e6-810d-eadb8a0d049a.png) and  
![image](https://cloud.githubusercontent.com/assets/6580269/23173678/96396926-f81f-11e6-8ba7-bae18025f90a.png)
- I don't know why, but I don't like these as much as their ribbon counterparts, and would prefer if we just kept the ribbon. 
![image](https://cloud.githubusercontent.com/assets/6580269/23173724/b9691c02-f81f-11e6-9744-e17ba519d030.png)
- what's up with the blue background onion?


I think Isabela would love to see the progress of this work and talk about it, all three of us. I'll ask her when she is free today during our meeting and work on setting up a meeting with all three of us. She is very pumped about these images and was the one championing it, and I bet showing these to her would make her happy!

**Comment by elioqoshi** [comment]
Hey Linda. Thanks for the kind words. Addressing your points:

- We need inverted images as a secondary option when the background where the logo/graphic is, is dark. You wouldn't put a purple Tor logo on black background, wouldn't you? So it needs to exists as an alternative always. Same with the badges really. Someone who has a dark themed website, would prefer to have a light badge.

- Regarding the support Tor buttons: I thought that we might need versions for smaller resolutions (the ribbons would not work on very small dimensions). We can leave these out, but we might want to have an alternative for smaller I support Tor buttons.

- The onion icon with the blue background is my proposal for the Tor Browser icon. My thinking behind that was that the blue circle would be seen as a globe. We can use another color here, but I'd rather think it properly through why we are using a certain color.

I'd be happy to discuss progress on a call :)

**Comment by lindanlee** [comment]
Hey! 

>We need inverted images as a secondary option when the background where the logo/graphic is, is dark. You wouldn't put a purple Tor logo on black background, wouldn't you? So it needs to exists as an alternative always. Same with the badges really. Someone who has a dark themed website, would prefer to have a light badge.

Oh, I didn't think of this. Good point. I am totally on board with these now. 

>Regarding the support Tor buttons: I thought that we might need versions for smaller resolutions (the ribbons would not work on very small dimensions). We can leave these out, but we might want to have an alternative for smaller I support Tor buttons.

Again, you have more experience with this, and I see your reasoning now! 

It was really nice to hear these rationalizations. I don't know if they are obvious, but it may not be to some people. Quick annotations on why you made each image (i.e. "for dark backgrounds") and when each should be used would be helpful (not just for me, but for people that will access the image repo). 

>= The onion icon with the blue background is my proposal for the Tor Browser icon. My thinking behind that was that the blue circle would be seen as a globe. We can use another color here, but I'd rather think it properly through why we are using a certain color.

I see. It does look nice. Again, rebranding, although not technically, is something I am treading likely. But I do like it. That is all I will say. 

I'll send out an event invite soon~ look out for it!

**Comment by elioqoshi** [comment]
Yeah, you are right, need to get out of my bubble sometimes and explain my reasoning behind the process more. Although it's good to see unbiased reactions by people just by looking at the results and continue a discussion on it afterwards (like we just did). This is a rather abstract discussion topic so let's move on with the work yep!

**Comment by elioqoshi** [comment]
@lindanlee Finished everything apart the Tor Metrics logo and the Slide Templates. I need some more feedback there. Pushed all the other assets here and should be good to go.

I will add the style guide assets (at least those which were not part of the assets here) in my free time some time before Amsterdam

**Comment by lindanlee** [comment]
@elioqoshi Thanks for the great work! 

Metrics logo: after some discussion with Scout and Isa during the style guide, we have decided that it's okay to let certain teams have their own identity, and to keep the metrics logo as is for now. 

Slide templates: what would you like to know? I am thinking of something minimal, with the tor-logo visible, that uses the Tor colors and fonts. 

Thanks for doing the style guide assets as well!

**Comment by Klusterfuk90** [comment]
> Hey there,
> 
> So I worked all day on the assets today and was able to finish a good chunk of it. This is where I am:
> 
> ![preview-01](https://cloud.githubusercontent.com/assets/5436686/23096934/db3c7e18-f627-11e6-8b43-ff13fa4e1258.png)
> 
> I will create different color variations for most of the assets, in case the classic purple doesn't cut it. In addition, I have added some various sizes for buttons which might come in handy.
> 
> I will have to do the Presentation slide templates yet, but that next week probably.
> 
> Please ask if you have any questions! 
> 
> cc @lindanlee

**Comment by Klusterfuk90** [comment]
AAAADMUoD7p/rciOwdTc4nC86x6rh444vOUJgyD9vdTjWrsVjXF7tOZiASWf9Po6HWarauxaAIWU3McV7iBBbOH7zwenG9NAOKo03bvrxauUrvM

**Comment by Klusterfuk90** [comment]
Why all the valuation
